cmd/mta.go

```diff
@@ -64,12 +58,13 @@ var copyCmd = &cobra.Command{
 	Long:  "Copy from source path to target path",
 	Args:  cobra.NoArgs,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		logs.Logger.Info("copy from source path: " + copyCmdSourcePath + " to target path: " + copyCmdTargetPath)
-		err := mta.CopyFile(copyCmdSourcePath, copyCmdTargetPath, os.Create)
-		if err != nil {
-			logs.Logger.Error(err)
-		}
-		return err
+		return mta.RunAndWriteResultAndHash(
+			"copy from source path: "+copyCmdSourcePath+" to target path: "+copyCmdTargetPath,
```

Comment:
I suggest to use templates instead of concatenations, it will also allow to move messages easier to constants